vue3-apollo-quasar/src/views/Repository/RepositoryDetails.vue

```diff
@@ -1,6 +1,5 @@
 <template>
   <q-page v-if="!loading">
-    <!-- TODO: Instead of using currentRepo?.data.isPrivate to determine visibility tag, use the actual visibility tag from GitHub (there's more than just "Public" & "Private". We also have "Internal" and potentially others)  -->
     <RepoSubHeader
       :username="owner"
       :repoName="repo"
```

Comment:
Another thing, in some places you use props with `camelCase` and in others with `kebab-case`. The common convention in vue is to use `kebab-case`. Could you rename all `camelCase` usages to `kebab-case`?